vico/compose/src/main/java/com/patrykandpatrick/vico/compose/cartesian/CartesianChart.kt

```diff
@@ -52,7 +52,7 @@ public fun rememberCartesianChart(
   bottomAxis: Axis<Axis.Position.Horizontal.Bottom>? = null,
   marker: CartesianMarker? = null,
   markerVisibilityListener: CartesianMarkerVisibilityListener? = null,
-  layerPadding: CartesianLayerPadding = cartesianLayerPadding(),
+  getLayerPadding: ((ExtraStore) -> CartesianLayerPadding) =  { CartesianLayerPadding() },
```

Comment:
Please use the compose factory function that was used before.
```suggestion
  getLayerPadding: ((ExtraStore) -> CartesianLayerPadding) =  { cartesianLayerPadding() },
```